corehq/apps/users/models.py

```diff
@@ -3282,6 +3282,14 @@ class ConnectIDUserLink(models.Model):
     connectid_username = models.TextField()
     commcare_user = models.ForeignKey(User, related_name='connectid_user', on_delete=models.CASCADE)
     domain = models.TextField()
+    messaging_consent = models.BooleanField(default=False)
 
     class Meta:
         unique_together = ('domain', 'commcare_user')
+
+
+class ConnectIDMessagingKey(models.Model):
+    domain = models.TextField()
```

Comment:
I don't think this is necessary since the `user_link` has a domain field already

Reply:
This should have an `active` boolean as well, so we rotate out keys

Reply:
I added the domain here to support the domain deletion test. It was complaining before. I will add the `active` boolean here as well.

Reply:
Added. https://github.com/dimagi/commcare-hq/pull/35212/commits/7d02e73e1f631a5182076b8f4ca41e4f8c585d68